### ISSUE TITLE
it's not true! haha

### ISSUE THREAD

**Issue by kotchuprik** [narze/THIS_REPO_HAS_1043_STARS#4]
![image](https://user-images.githubusercontent.com/802946/131750459-00edff24-6e85-456e-9705-235855de1393.png)

**Comment by FayasNoushad** [comment]
Its auto updating

**Comment by narze** [comment]
![image](https://user-images.githubusercontent.com/248741/131777042-018bcc02-770f-4089-9731-a2793f2f3ade.png)

**Comment by narze** [comment]
As I documented :

![image](https://user-images.githubusercontent.com/248741/131777094-8b2cc2da-74f2-493c-af5c-eaa9de239bcc.png)

**Comment by kotchuprik** [comment]
it works only in one direction :)

**Comment by narze** [comment]
Yeah, since Github API does not support `unstar` event, but we have the schedule action which will sync the star count every hour :)

**Comment by kotchuprik** [comment]
ah, gotcha

**Comment by avipars** [comment]
if i click and unclick and then click again... will that get me more... otherwise i guess you can scrape the site

**Comment by narze** [comment]
> if i click and unclick and then click again... will that get me more... otherwise i guess you can scrape the site

Nope, the Github action will read the actual star count by the time it runs.

I think I should add in readme about how this repo work?

**Comment by Polar-69** [comment]
![image](https://user-images.githubusercontent.com/81021823/131874471-30262773-30ed-4800-97ea-adb21c89f8f5.png)
shouldnt that be 13 somethign why 1269

**Comment by narze** [comment]
> ![image](https://user-images.githubusercontent.com/81021823/131874471-30262773-30ed-4800-97ea-adb21c89f8f5.png)
> shouldnt that be 13 somethign why 1269

It was rounded up. I can't do anything with that.

**Comment by narze** [comment]
I just added the [Q&A wiki](https://github.com/narze/THIS_REPO_HAS_X_STARS/wiki/Q&A) if you are interested.

**Comment by Polar-69** [comment]
> > ![image](https://user-images.githubusercontent.com/81021823/131874471-30262773-30ed-4800-97ea-adb21c89f8f5.png)
> > shouldnt that be 13 somethign why 1269
> 
> It was rounded up. I can't do anything with that.

ye its ok im jsut sayign it for fun but idk why ppl getting mad that its wrong it will take some time to update duh they dont get it maybe but ye good work
